Makefile

```diff
@@ -87,12 +87,15 @@ $(GOTMP)/bin/completions.tar.gz: build
 
 mkcert: $(GOTMP)/bin/darwin_arm64/mkcert $(GOTMP)/bin/darwin_amd64/mkcert $(GOTMP)/bin/linux_arm64/mkcert $(GOTMP)/bin/linux_amd64/mkcert
 
+# Set CURL to the Homebrew-installed curl, fallback to default
+CURL := $(shell command -v /opt/homebrew/opt/curl/bin/curl || command -v /usr/local/opt/curl/bin/curl || echo curl)
```

Comment:
```suggestion
CURL := $(shell command -v /opt/homebrew/opt/curl/bin/curl || command -v /usr/local/opt/curl/bin/curl || command -v curl)
```

Reply:
This should be exactly the same, the `echo curl` will make CURL=curl, which will get curl from PATH. Your change is perfectly valid, will give the full path to the curl in the PATH instead of just `curl`